app/_includes/footer.html

```diff
@@ -14,21 +14,24 @@
           <li><a href="/about/#team-history">History</a></li>
           <li><a href="/about/#robots">Robots</a></li>
           <li><a href="/about/sponsors/">Sponsors</a></li>
+          <li><a href="/about/people/">People</a></li>
         </ul>
       </li>
       <li>
-        <a href="/about/people">Members</a>
+        <a href="/resources/">Resources</a>
         <ul>
           <li><a href="/members/calendar/">Calendar</a></li>
-          <li><a href="/about/people/">People</a></li>
+          <li><a href="/resources/links/">Helpful Links</a></li>
         </ul>
       </li>
       <li>
-        <a href="/resources/">Resources</a>
+        <h6>SOCIAL MEDIA</h6>
```

Comment:
outside scope - this could cause some merge conflcits with #183 

hint: to commit more relevant stuff only, use `git add -p` to interactively stage select patches only